go-controller/pkg/config/config.go

```diff
@@ -199,6 +200,7 @@ type KubernetesConfig struct {
 	OVNEmptyLbEvents     bool   `gcfg:"ovn-empty-lb-events"`
 	PodIP                string `gcfg:"pod-ip"` // UNUSED
 	RawNoHostSubnetNodes string `gcfg:"no-hostsubnet-nodes"`
+	EgressIPEnabled      bool   `gcfg:"egress-ip-enable"`
```

Comment:
`KubernetesConfig` is supposed to be for stuff related to kubernetes itself. It should not be used for enabling/disabling ovn-kubernetes features. It is possible that some of the existing config options get this wrong but we shouldn't be making it worse. (Possibly need `OVNKubenetesFeatureConfig` or something?)